cli/cmd/main.go

```diff
@@ -218,7 +218,7 @@ func main() {
 					{
 						Name:  "export",
 						Usage: "download data from Viam cloud",
-						UsageText: fmt.Sprintf("viam data <%s> <%s> [%s] [%s] [%s] [%s] [%s] [%s] [%s] [%s] [%s] [%s] [%s] [%s] [%s] [%s]",
```

Comment:
I believe the <> versus [] was for required versus optional

Reply:
Ah thanks for the catch! Updated back.